app/scripts/modules/kubernetes/cluster/configure/CommandBuilder.js

```diff
@@ -137,8 +137,10 @@ module.exports = angular.module('spinnaker.kubernetes.clusterCommandBuilder.serv
     }
 
     function buildImageId(image) {
-      if (image.fromContext) {
+      if (image.fromContext && image.pattern && image.cluster) {
```

Comment:
Does this mean it came from a findImage? I think we should be more explicit here